web_responsive/static/src/less/main.less

```diff
@@ -4,6 +4,7 @@
 body {
     width: 100%;
     height: 100%;
+    overflow: hidden!important; // Important is to override base template
```

Comment:
Apparently this `style` attribute is being added by The `drawer.css` library because `<body>` has a `drawer` class, added by the `web_responsive` module. We're still investigating.